### ISSUE TITLE
Phing default target fails with ConfigurationException

### ISSUE THREAD

**Issue by ulrikkold** [fabarea/t3phing#6]
I went through the Wizard (which was a revelation to me - I didn't know you could do that with Phing) and tried running the default target ('help'), but all I got was this ~~lousy t-shirt~~ error message: 

$> bin/phing
Buildfile: /home/uhk/Projects/vagrant-aal/build.xml
[PHP Error] include_once(fetchPropertiesTask.php): failed to open stream: No such file or directory [line 1224 of /home/uhk/Projects/vagrant-aal/Packages/Libraries/phing/phing/classes/phing/Phing.php]
[PHP Error] include_once(): Failed opening 'fetchPropertiesTask.php' for inclusion (include_path='/home/ulrikkold/Projects/testproject/build/Phing/Classes:/home/ulrikkold/Projects/testproject/Packages/Libraries/phing/phing/bin/../classes:/home/ulrikkold/Projects/testproject/Packages/Libraries/phpunit/php-text-template:/home/ulrikkold/Projects/testproject/Packages/Libraries/phpunit/php-timer:/home/ulrikkold/Projects/testproject/Packages/Libraries/phing/phing/classes') [line 1224 of /home/ulrikkold/Projects/testproject/Packages/Libraries/phing/phing/classes/phing/Phing.php]
       [if] Error in IfTask

BUILD FAILED
/home/ulrikkold/Projects/testproject/build.xml:16:5: Error in IfTask
(stack trace goes here.)

Previous exception 'ConfigurationException' with message 'Error importing fetchPropertiesTask.php' in /home/ulrikkold/Projects/testproject/Packages/Libraries/phing/phing/classes/phing/Phing.php:1232
(stack trace goes here)

I installed a local build toolchain (phploc, phpmd, phpcs, phing, etc.) and run phing from ./bin/phing instead of using a global install - but that shouldn't be a problem, I guess?

I have tried both phing v2.8.2 and v2.10.1 - but the problem persists. Are you familiar with a phing version requirement for using your repo?

**Comment by fabarea** [comment]
> I installed a local build toolchain (phploc, phpmd, phpcs, phing, etc.) and run phing from ./bin/phing instead of using a global install - but that shouldn't be a problem, I guess?

It shouldn't. You could try nevertheless downloading the phing.par executable and run it from the the root directory where the build.xml file stands.

How does your tree looks like? Shortly put:

```
├── build
│   └── Phing  <-- the code of this repository repository
├── build.xml  <-- the main build file which is pretty much empty, only holds the includes. !!! phing must be run at this level !!!
├── configuration
│   ├── PhingSettings.properties <-- file will be created when retrieving database credentials
│   ├── PhingSettings.xml  <-- file will be created at the end of the wizard
├── htdocs  <-- website

```

**Comment by ulrikkold** [comment]
The structure looks approx. like this. There are some minor differences which should not actually make difference.

However, after searching for ```fetchPropertiesTask``` import, I figured that there is a typo in build/Phing/Master.xml.

```
>$ grep -rn 'fetchPropertiesTask' *
build/Phing/Master.xml:48:	<taskdef name="fetchProperties" classname="fetchPropertiesTask" classpathref="t3phing.classes"/>
```

I renamed that classname property from 'fetchPropertiesTask' to 'FetchPropertiesTask', which is the actual name of the class in build/Phing/Classes. That helped a lot!

I consider this issue closed.
